python/packaging/connect/setup.py

```diff
@@ -101,6 +101,7 @@
         "pyspark.pandas.tests.connect.groupby",
         "pyspark.pandas.tests.connect.indexes",
         "pyspark.pandas.tests.connect.io",
+        "pyspark.pandas.tests.connect.pandas",
```

Comment:
Shouldn't this be: `pyspark.sql.tests.connect.pandas`?

Reply:
yes :-). making a pr now

Reply:
https://github.com/apache/spark/pull/49119